tests/test_export_rms/test_export_rms_volumetrics.py

```diff
@@ -14,6 +14,7 @@
 
 
 VOLDATA = (Path("tests/data/drogon/tabular/geogrid--vol.csv")).absolute()
+CONFIG_PATH = Path("../../fmuconfig/output/global_variables.yml")
```

Comment:
Do you think it make sense to import `from fmu.dataio.export.rms._utils import CONFIG_PATH` instead? I don't expect the _path_ to change, but maybe (or maybe not) it adds a bit to the test

Reply:
Originally had that.. but got an import error from the `import_rms_package` (that are run when importing utils) as it expects to be in RMS. But will give it a slight rewrite to avoid this 👍